cmd/config-host-main.go

```diff
@@ -275,14 +283,22 @@ func listHosts() {
 			maxAlias = len(k)
 		}
 	}
+	var hosts []hostMessage
+	sort.Sort(byAlias(hosts))
 	for k, v := range conf.Hosts {
-		printMsg(hostMessage{
+
+		hosts = append(hosts, hostMessage{
 			op:        "list",
-			Alias:     fmt.Sprintf("%*.*s", maxAlias, maxAlias, k),
+			Alias:     fmt.Sprintf("%-*.*s", maxAlias, maxAlias, k),
```

Comment:
You don't need to do it twice.. Do it once below and here keep it as 'Alias' itself. So that we can sort it properly.